src/pip/_internal/operations/install/wheel.py

```diff
@@ -412,6 +412,16 @@ def _raise_for_invalid_entrypoint(specification: str) -> None:
 
 
 class PipScriptMaker(ScriptMaker):
+    # Override distlib's default script template with one that
+    # doesn't import `re` module, allowing scripts to load faster.
+    script_template = r"""import sys
```

Comment:
nit: I'd prefer we do a `textwrap.dedent` for this template, just to make it easier to work with non-syntax aware collapses (and align the visual hierarchy).

Reply:
:+1: added `textwrap.dedent` and made the r-string a regular string that has a backslash escape for the initial newline.